Code/Tools/SceneAPI/SceneUI/RowWidgets/TransformRowWidget.cpp

```diff
@@ -103,12 +103,12 @@ namespace AZ
             TransformRowWidget::TransformRowWidget(QWidget* parent)
                 : QWidget(parent)
             {
-                QWidget* hider = new QWidget();
```

Comment:
If you parent `hider` to `this` I think you can omit the rest of the changes, since it will get deleted when `TransformRowWidget` gets deleted. It doesn't look like the `hider` widget is getting deleted/re-created whenever the custom row widget is returned to the pool, it should just be a one-time initialization.

Reply:
I though the layout would change the owning parent.

Reply:
I just tried this today, @cgalvan the parent is changed when you add it to the layout.

Reply:
I was originally going to say that this should just get cleaned up normally by the layout taking ownership, but I just realized the `TransformRowWidget` is actually modifying the layout of its parent, which is pretty yucky. But I think re-working that is outside the scope of this, so I think this is OK for now. We should look at this deeper in the future though @alexmontAmazon @dtamkin1